--- a/setup.py
+++ b/setup.py
@@ -12,19 +12,6 @@
 
 
 def _compile_ccl(debug=False):
-    call(["mkdir", "-p", "build"])
-    v = sys.version_info
-    cmd = ["cmake", "-H.", "-Bbuild",
-           "-DPYTHON_VERSION=%d.%d.%d" % (v.major, v.minor, v.micro)]
-    if debug:
-        cmd += ["-DCMAKE_BUILD_TYPE=Debug"]
-    if call(cmd) != 0:
-        raise Exception(
-            "Could not run CMake configuration. Make sure "
-            "CMake is installed !")
-
-    if call(["make", "-Cbuild", "_ccllib"]) != 0:
-        raise Exception("Could not build CCL")
 
     # Finds the library under its different possible names
     if os.path.exists("build/pyccl/_ccllib.so"):
